saleor/attribute/__init__.py

```diff
@@ -22,6 +22,12 @@ class AttributeInputType:
     # list of the input types that can be used in variant selection
     ALLOWED_IN_VARIANT_SELECTION = [DROPDOWN]
 
+    TYPES_WITHOUT_CHOICES = [
+        REFERENCE,
+        NUMERIC,
+        RICH_TEXT,
+    ]
```

Comment:
Shouldn't this work the other way around, listing the few types where we want to return choices?

Reply:
Fine by me